logstreams/src/test/java/io/camunda/zeebe/logstreams/log/LogStreamBatchWriterTest.java

```diff
@@ -115,80 +113,40 @@ public void shouldReturnPositionOfLastEvent() {
   @Test
   public void shouldWriteEventWithValueBuffer() {
     // when
-    final long position =
-        writer.tryWrite(
-            new MutableLogAppendEntry().key(1).recordValue(EVENT_VALUE_1),
-            new MutableLogAppendEntry().key(2).recordValue(EVENT_VALUE_2));
-
-    // then
-    final List<LoggedEvent> events = getWrittenEvents(position);
-    assertThat(getValueBuffer(events.get(0))).isEqualTo(EVENT_VALUE_1);
-    assertThat(getValueBuffer(events.get(1))).isEqualTo(EVENT_VALUE_2);
-  }
-
-  @Test
-  public void shouldWriteEventWithValueBufferPartially() {
```

Comment:
:thought_balloon: I removed this test case because it doesn't really apply anymore. The logstream will only write `UnifiedRecordValue`, this was testing writing part of a buffer.